examples/otlp/docker/docker-compose.yaml

```diff
@@ -26,8 +26,7 @@ services:
       - "8888:8888"   # Prometheus metrics exposed by the collector
       - "8889:8889"   # Prometheus exporter metrics
       - "13133:13133" # health_check extension
-      - "55678"       # OpenCensus receiver
-      - "55681:55679" # zpages extension
```

Comment:
There is no open census receiver configured, so no need for `55678`. 

`55681` is [used](https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/factory.go#L35) as the legacy oltp http port. Better to just use `55679` for zpages.